Shaders/water_pass/water_pass.frag.glsl

```diff
@@ -9,10 +9,17 @@ uniform sampler2D tex;
 uniform sampler2D sbase;
 uniform sampler2D sdetail;
 uniform sampler2D sfoam;
+uniform mat4 P;
+uniform mat3 V3;
```

Comment:
Last one - please add `#ifdef _SSR` here, not necessary but helps to tell it's for the SSR feature.